sqlglot/__init__.py

```diff
@@ -1,6 +1,6 @@
 from sqlglot.dialects import Dialect
 from sqlglot.errors import ErrorLevel, UnsupportedError, ParseError, TokenError
-from sqlglot.expressions import Expression, Select
+from sqlglot.expressions import Expression, Select, And, Or, Not, and_, or_, condition
```

Comment:
why do you need to export And Or

Reply:
Good catch. I originally had `and_`, `or_`, etc in this file, but moved 'em down so I can use them within the expression module.